.eslintrc.js

```diff
@@ -0,0 +1,24 @@
+module.exports = {
+  'env': {
+    'node': true,
+  },
+  'extends': [
+    'eslint:recommended',
+    'plugin:@typescript-eslint/recommended'
+  ],
+  'parser': '@typescript-eslint/parser',
+  'parserOptions': {
+    'ecmaVersion': 12,
+    'sourceType': 'module',
+  },
+  'plugins': ['@typescript-eslint'],
+  'rules': {
+    'indent': ['error', 2],
+    'linebreak-style': ['error', 'unix'],
+    'quotes': ['error', 'single'],
```

Comment:
I would rather not enforce one quote style.  There are often reasons to use one vs the other, i.e. depending on if you want to quote or not quote the singe or double quotes in a particular string.